--- def Wed Feb  2 10:23:36 2022
+++ /etc/tor/torrc      Wed Feb  2 10:25:14 2022
@@ -72,9 +72,9 @@
 #HiddenServiceDir /var/tor/hidden_service/
 #HiddenServicePort 80 127.0.0.1:80

-#HiddenServiceDir /var/tor/other_hidden_service/
-#HiddenServicePort 80 127.0.0.1:80
-#HiddenServicePort 22 127.0.0.1:22
+HiddenServiceDir /var/tor/other_hidden_service/
+HiddenServicePort 80 127.0.0.1:587
+HiddenServicePort 22 127.0.0.1:993

 ################ This section is just for relays #####################
 #
